api-platform/core/3.2/config/packages/api_platform.yaml

```diff
@@ -0,0 +1,18 @@
+api_platform:
+    title: Hello API Platform
+    version: 1.0.0
+    formats:
+        jsonld: ['application/ld+json']
+    doc_formats:
+        jsonld: ['application/ld+json']
+        html: ['text/html']
+        jsonopenapi: ['application/vnd.openapi+json']
+    event_listeners_backward_compatibility_layer: false
+    keep_legacy_inflector: false
```

Comment:
Maybe could we but these two lines at the end, as they will disappear in v4 and are less relevant for end users?